18831.xml

```diff
@@ -52,7 +52,7 @@ https://ioterop.com/
                 <Type>Corelnk</Type>
                 <RangeEnumeration></RangeEnumeration>
                 <Units></Units>
-                <Description><![CDATA[The source of the data to publish. If this Resource is empty, the published data are implementation dependent.]]></Description>
+                <Description><![CDATA[The source of the data to publish (e.g. "</sensors/temp>", or "</3303/0/5700>;</3336/0>"). If this Resource is empty, the published data are implementation dependent.]]></Description>
```

Comment:
Should there still be more details how to interpret the CoRE link in this context? Something link "the client sends the information referenced by the link when ever it changes" (with some notion of change)?